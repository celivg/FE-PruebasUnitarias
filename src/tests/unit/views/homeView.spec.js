import { describe, it, expect } from "vitest";
import { mount } from "@vue/test-utils";
import HomeView from "@/views/HomeView.vue";

describe("HomeView", () => {
  const wrapper = mount(HomeView);

  it('La estructura HTML se mantiene estable', () => {
    expect(wrapper.html()).toMatchSnapshot();
  })
});