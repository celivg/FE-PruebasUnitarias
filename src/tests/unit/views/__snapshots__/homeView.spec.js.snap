// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView > La estructura HTML se mantiene estable 1`] = `
"<div class="home"><img alt="Vue logo" src="/src/assets/logo.png">
  <div class="posts">
    <h1>Vista de Posts</h1>
    <ul>
      <li>Post 1</li>
      <li>Post 2</li>
      <li>Post 3</li>
      <li>Post 4</li>
    </ul>
  </div>
</div>"
`;
